_targets/build.common

```diff
@@ -60,16 +63,16 @@ b_mkscript_user() {
 
 					PROGS+=("$name")
 					psz=$(wc -c < "${PREFIX_PROG_STRIPPED}$name")
-					printf "alias %s 0x%x 0x%x\n" "$name" "$poffs" "$psz"
+					printf "alias%s %s 0x%x 0x%x\n" "$RELATIVE_ALIAS" "$name" "$poffs" "$psz"
```

Comment:
I would set RELATIVE_ALIAS to "-r" inside this function, not outside to avoid hacking possibilities :) 
Also you shouldn't rely on not setted variables in terms of purity (you can explicitly set it to empty string if not used)

Same applied to preinit